flutter_app/pubspec.lock

```diff
@@ -423,6 +423,14 @@ packages:
       url: "https://pub.dev"
     source: hosted
     version: "1.1.0"
+  web:
+    dependency: transitive
+    description:
+      name: web
+      sha256: dc8ccd225a2005c1be616fe02951e2e342092edf968cf0844220383757ef8f10
```

Comment:
Quotation marks here as well. Just to maintain consistency.

Reply:
This is an automatically generated file.